src/bp-activity/classes/class-bp-activity-follow.php

```diff
@@ -176,12 +188,20 @@ public static function get_followers( $user_id ) {
 	 *
 	 * @since BuddyBoss 1.0.0
 	 *
-	 * @param int $user_id The user ID to fetch.
+	 * @param int   $user_id    The user ID to fetch.
+	 * @param array $query_args Query arguments.
+	 * 
```

Comment:
```suggestion
	 * @since BuddyBoss [BBVERSION] added support for query arguments.
	 *
	 * @param int   $user_id    The user ID to fetch.
	 * @param array $query_args Query arguments.
	 *
```